realtime/redisSubscriber.js

```diff
@@ -24,11 +26,30 @@ module.exports = (io) => {
     // message object to be sent to the clients
     const mssgObj = JSON.parse(mssgStr);
     const key = Object.keys(mssgObj)[0];
+    const parsedObj = rtUtils.parseObject(mssgObj[key], key);
+    if (featureToggles.isFeatureEnabled('publishPartialSample') &&
+    rtUtils.isThisSample(parsedObj)) {
+      const useSampleStore =
+        featureToggles.isFeatureEnabled('enableRedisSampleStore');
 
-    /*
-     * pass on the message received through the redis subscriber to the socket
-     * io emitter to send data to the browser clients.
-     */
-    emitter(io, key, mssgObj);
+      // assign the subject db model if sampleStore is not enabled
+      const subjectModel =
+        useSampleStore ? undefined :
+          require('../db/index').Subject; // eslint-disable-line global-require
+      rtUtils.attachAspectSubject(parsedObj, useSampleStore, subjectModel)
```

Comment:
we can conditionally pass in the subjectModel, correct? If sampleStore is off, pass in subjectModel, else don't pass it in. Thus we don't need the useSampleStore param